### DISCUSSION TITLE
kubeflow pipeline version and  kfp sdk version

### DISCUSSION THREAD

**Discussion by lanzhiwang** [kubeflow/pipelines#10852] [General]
We have deployed different versions of kubeflow pipeline in different kubernetes environments. I would like to know which version of kfp sdk version should be used for different versions of kubeflow pipeline and where can I find relevant information?

**Comment by rimolive** [comment]
Let me know if that link helps: https://www.kubeflow.org/docs/components/pipelines/v2/migration/#terminology
